Paccy10/ampersand_app_api
# Feature(drivers): create driver

**What does this PR do?**

- Create driver

**Description of Task to be completed?**

- Register a new driver

**How should this be manually tested?**

- Checkout to branch `ft-create-driver`
- Run the app with `flask run`
- Send a `POST` request to `/drivers` with the required payload
- A new driver record should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
